app/models/ServiceCall.scala

```diff
@@ -2,12 +2,32 @@ package models
 
 import com.amazonaws.services.dynamodbv2.document.Item
 import org.cvogt.play.json.Jsonx
-import play.api.libs.json.{Format, JsValue, Json}
+import play.api.libs.json._
+
+case class ServiceData(
```

Comment:
Might be a good idea to make all the parameters default to 'None', so things don't fall over if they aren't provided